lang/query/src/info/data.rs

```diff
@@ -183,6 +190,9 @@ impl From<CallInfo> for InfoContent {
 /// Information for dotcalls (destructors or definitions)
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct DotCallInfo {
+    /// The span where the dotcall was defined.
+    /// This is used for the jump-to-definition feature.
+    pub target_span: Option<(Url, Span)>,
```

Comment:
Maybe `definition_site` is a bit clearer than `target_span`?

Reply:
Done